src/backend/schemas/agent.py

```diff
@@ -93,6 +100,7 @@ class CreateAgentRequest(BaseModel):
     # deployment_id or deployment_name
     deployment: str
     organization_id: Optional[str] = None
+    is_private: Optional[bool] = False
```

Comment:
Do we need to add this to the update schema?

Reply:
Will add updating agents in a separate PR!

Reply:
Thanks